migrations/default/1_create_tables/up.sql

```diff
@@ -59,14 +119,114 @@ create table db.crashes_vz (
     crash_id integer primary key not null references db.crashes_cris (
         crash_id
     ) on delete cascade on update restrict,
-    road_type_id integer references db.road_types (id)
+    road_type_id integer references db.road_types (id),
+    onsys_fl boolean,
+    rr_relat_fl boolean,
+    schl_bus_fl boolean,
+    toll_road_fl boolean,
+    private_dr_fl boolean,
+    txdot_rptable_fl boolean,
+    thousand_damage_fl boolean,
+    road_constr_zone_fl boolean,
+    medical_advisory_fl boolean,
+    active_school_zone_fl boolean,
+    road_constr_zone_wrkr_fl boolean,
+    at_intrsct_fl boolean,
+    case_id text,
+    rpt_block_num text,
+    rpt_sec_block_num text,
+    rpt_sec_hwy_num text,
+    rpt_sec_street_desc text,
+    rpt_sec_street_name text,
+    rpt_sec_street_pfx text,
+    rpt_sec_street_sfx text,
+    rpt_street_desc text,
+    rpt_street_name text,
+    rpt_street_pfx text,
+    rpt_street_sfx text,
+    rpt_hwy_num text,
+    rpt_ref_mark_dir text,
+    rpt_ref_mark_dist_uom text,
+    surf_type_id integer,
+    crash_date timestamp with time zone,
+    latitude numeric,
+    longitude numeric,
+    rpt_ref_mark_offset_amt numeric,
+    crash_speed_limit integer,
+    fhe_collsn_id integer,
+    light_cond_id integer,
+    obj_struck_id integer,
+    rpt_rdwy_sys_id integer,
+    rpt_road_part_id integer,
+    rpt_sec_rdwy_sys_id integer,
+    rpt_sec_road_part_id integer,
+    traffic_cntl_id integer,
+    wthr_cond_id integer,
+    intrsct_relat_id integer,
+    investigat_agency_id integer,
+    surf_cond_id integer,
+    rpt_city_id integer,
+    rpt_cris_cnty_id integer,
+    investigator_narrative text,
+    crash_time timestamp with time zone
```

Comment:
should the shape of the vz table looks exactly like the shape of the cris table ie. having all the same columns/column names? if so, maybe thats something we should enforce like with a constraint? idk if that would be helpful. but i imagine all the extra custom columns we will put in the unified table right?

Reply:
i think it should be really close to identical, although i think we should definitely add audit fields. i'm not sure that we can enforce column names with a constraint, but i am definitely envisioning that the column metadata in `columns.py` becomes our guide for keeping track of CRIS vs non-cris columns. putting this info the DB rather than a random file might be the best bet.

Reply:
i like where y'all are going with this, and another advantage of keeping the metadata in the DB is that it could be used in the VZE app or other processes. I'm thinking about config like [the editable key in crashDataMap](https://github.com/cityofaustin/atd-vz-data/blob/34066e5d3c79a479c5c2503106d38dd6439f4d09/atd-vze/src/views/Crashes/crashDataMap.js#L55) and maybe even what lookup table is used by a column. There is more to it, but y'all made me start thinking about ways to centralize config.

Reply:
yeah!